client/js/screens/AuthLoadingScreen/AuthLoadingScreenContainer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  StatusBar,
+  StyleSheet,
+  View,
+  Text,
+} from 'react-native';
+import AsyncStorage from '@react-native-community/async-storage';
+import Loader from '../../components/Loader';
+
+export default class AuthLoadingScreenContainer extends React.Component {
+  constructor() {
+    super();
+    this._bootstrapAsync();
+  }
+
+  // Fetch the token from storage then navigate to our appropriate place
+  _bootstrapAsync = async () => {
+    const userToken = await AsyncStorage.getItem('userToken');
```

Comment:
if you are getting useToken, you should be setting it somewhere too. Please use Async storage to store your user's token when they login or signup.

Reply:
Done